tests/suites/casper/formfill.js

```diff
@@ -1,5 +1,6 @@
 /*eslint strict:0*/
 var fs = require('fs');
+var x = require('casper').selectXPath;
 
```

Comment:
Ok, never mind. :-) This is just a personal preference (it is easier to e.g. ```grep``` for variables with more than one letter). You decide.